lib/create/index.js

```diff
@@ -0,0 +1,73 @@
+const {
+  GitHubUser,
+  Subscription,
+} = require('../models');
+
+const createIssueDialog = require('../messages/create/create-issue-dialog');
+
+async function openDialog(req, res) {
+  const {
+    command,
+    gitHubUser,
+    slackWorkspace,
+  } = res.locals;
+
+  const subscriptions = await Subscription.findAll({
+    where: { channelId: req.body.channel_id, slackWorkspaceId: slackWorkspace.id },
+  });
+  if (subscriptions.length === 0) {
+    return command.respond({ text: 'This channel is not subscribed to any repositories' });
+  }
+
+  let repositories = await Promise.all(subscriptions.map(async (subscription) => {
+    const github = gitHubUser.client;
+    try {
+      const repository = await github.repos.getById({ id: subscription.githubId });
+      return repository.data;
+    } catch (err) {
+      req.log.error({ err, repoId: subscription.githubId }, 'Could not find repository for subscription');
+      if (err.code !== 404) {
+        throw err;
+      }
+    }
+  }));
+  // remove undefined
+  repositories = repositories.filter(repo => repo);
```

Comment:
I think we're fetching a list of repositories elsewhere. Can we move this logic into a reusable module somewhere that takes a client and a list of repositories IDs and returns the repos?